compiler/src/iree/compiler/Codegen/Common/CPU/CPUMaterializeEncodingPass.cpp

```diff
@@ -159,7 +159,17 @@ enumerateMatmulTileArm64(TypeRange elementTypes, ExecutableTargetAttr target) {
 
   if (!hasUkernel(target)) {
     if (lhs.isSignlessInteger(8) && rhs.isSignlessInteger(8) &&
-        (out.isSignlessInteger(32) || out.isF32())) {
```

Comment:
what happens to f32?

Reply:
This is probably my fault as I copied these tile sizes from the ukernel version. We can restore it and revisit separately to make sure any changes to the f32 cases don't add any noise. We shouldn't be using i8mm tile sizes for f32 as i8mm is not used for those cases. We want the f32 cases to fall into the default case.

Reply:
I see, @KoolJBlack can you add it to the PR description which explains why f32 is removed?

Reply:
done